setup.py

```diff
@@ -535,6 +534,10 @@ def finalize_options(self):
                 "onnxruntime.training.utils",
                 "onnxruntime.training.utils.data",
                 "onnxruntime.training.utils.hooks",
+                "onnxruntime.training.api",
```

Comment:
Do you need to check the platform and include only the necessary modules? For example is Mac then skip *.ortmodule.*

Reply:
I think we can keep the package more or less the same across all the os'. ORTModule will still not be available because of the compile time function `is_ortmodule_available`.